autometa/main.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python
```

Comment:
In the link you provided they used `__main__.py` and here you use `main.py`. Is this a convention that must be followed?

Reply:
I can change this, I'm not familiar enough with packaging/distribution conventions to know one way or the other.